lib/denylist/http.go

```diff
@@ -31,15 +31,15 @@ func CollectionEndpoint(denylist *sync.Map) func(http.ResponseWriter, *http.Requ
 }
 
 // SingleEndpoint serves the endpoints for particular Denylist entries at /denylist/...
-func SingleEndpoint(denylist *sync.Map) func(http.ResponseWriter, *http.Request) {
+func SingleEndpoint(denylist *sync.Map, syncer *Syncer) func(http.ResponseWriter, *http.Request) {
 	return func(response http.ResponseWriter, request *http.Request) {
 		switch request.Method {
 		case "GET":
 			getDenylistEntry(response, request, denylist)
```

Comment:
eh, how does this work without syncer

Reply:
oh, nvm, this is held in memory

Reply:
https://github.com/tulip/oplogtoredis/blob/be000ea2dbea01d6651bcb4ca898a0d7b59f9517/lib/denylist/pg.go#L17
there's always a syncer it just doesn't do anything if there's no pg config